test/e2e/specs/onboarding/start__write.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * @group calypso-pr
+ */
+
+import {
+	DataHelper,
+	StartSiteFlow,
+	MyHomePage,
+	TestAccount,
+	EditorPage,
+} from '@automattic/calypso-e2e';
+import { Browser, Page } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Hero Flow' ), () => {
+	const blogName = DataHelper.getBlogName();
+	const tagline = `${ blogName } tagline`;
+	const theme = 'Quadrat Green';
+
+	let page: Page;
+	let startSiteFlow: StartSiteFlow;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+		startSiteFlow = new StartSiteFlow( page );
+
+		const testAccount = new TestAccount( 'defaultUser' );
```

Comment:
(Also, since potential changes from this could possibly be more substantive, no need to make any updates based on this comment -- I'm mostly just interested in your perspective! 🙂 )

Reply:
I like this concept since creating a new site at the start will exactly replicate the entire Hero Flow. Given that there are many moving parts, it's true that we might hit some weird artifacts configuring an existing site. It can be pretty expensive to maintain the rest of the tests if it breaks!

> (Also, since potential changes from this could possibly be more substantive, no need to make any updates based on this comment -- I'm mostly just interested in your perspective! 🙂 )

Sure! In the meantime, I will be updating the test to use `calypsoPreReleaseUser`.

Reply:
>I like this concept since creating a new site at the start will exactly replicate the entire Hero Flow. Given that there are many moving parts, it's true that we might hit some weird artifacts configuring an existing site. It can be pretty expensive to maintain the rest of the tests if it breaks!

Well said! 😄 

>In the meantime, I will be updating the test to use calypsoPreReleaseUser.

Looks good! In the interest of time (and since you've done such a nice job on this PR 🥳 ), I think we can wait on doing the other changes here for now. That we can get you moved on to stage 3 of the trial! 🙌